tests/test_operator.py

```diff
@@ -141,6 +141,71 @@ def test_diagonal(dtype, getkey):
         assert jnp.allclose(lx.diagonal(operator), matrix_diag)
 
 
+@pytest.mark.parametrize("dtype", (jnp.float64,))
+def test_Woodbury(dtype, getkey):
+    tol = 1e-4
+    N = 20
+    k = 2
+
+    A = jr.normal(getkey(), (N, N), dtype=dtype)
+    C = jr.normal(getkey(), (k, k), dtype=dtype)
+    U = jr.normal(getkey(), (N, k), dtype=dtype)
+    V = jr.normal(getkey(), (k, N), dtype=dtype)
+
+    # Full matrix A
+    WB = lx.WoodburyLinearOperator(lx.MatrixLinearOperator(A), C, U, V)
+
+    full_matrix = WB.as_matrix()
+
+    true_x = jr.normal(getkey(), (N,))
```

Comment:
```suggestion
    true_x = jr.normal(getkey(), (N,), dtype=dtype)
```
and in two other places too, and then you can add `complex128` to type parametrization